packages/kit/package.json

```diff
@@ -43,7 +43,7 @@
 		"vitest": "^0.31.0"
 	},
 	"peerDependencies": {
-		"svelte": "^3.54.0",
+		"svelte": "^3.54.0 || ^4.0.0-next.0",
```

Comment:
```suggestion
		"svelte": "^3.54.0 || ^4.0.0-next.0 || ^4.0.0",
```

Reply:
I tested and verified it's not necessary. but it does turn out that we need to add `^4.0.0-next.0` to `svelte-hmr`, so I'm going to mark this PR as draft

Reply:
Oh TIL, I thought that you need it separately

Reply:
Actually, I realized I tested the wrong thing. I still think it's not necessary, but we should confirm before merging this to make sure

Reply:
Yeah, we only need the one. Explained on https://www.npmjs.com/package/semver